packs/feats/youre-next.json

```diff
@@ -38,6 +38,31 @@
                 "selector": "intimidation",
                 "type": "circumstance",
                 "value": 2
+            },
+            {
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Feat.YoureNext.Trigger"
+                    },
```

Comment:
We can use a title of `Trigger` here, which will need localized, preferably in a generic place under specific rule. `SpecificRule.Formatting` or something like that?

Reply:
I believe we already have that, I just forgot to put it, will check

Reply:
We didn't. Put it under `PF2E.FeatDescription` which is already being used for things like these.